content/master/concepts/providers.md

```diff
@@ -62,34 +316,87 @@ spec:
       key: creds
 ```
 
-You can see that there is a reference to a key in a specific `Secret`. The value
-of that key should contain the credentials that the controller will use. The
-documentation of each provider should give you an idea of how that credentials
-blob should look like.
+{{< hint "important" >}}
+Authentication configuration may be different across Providers. 
 
-The following is an example usage of AWS `ProviderConfig`, referenced by a
-`RDSInstance`:
+Read the documentation on a specific Provider for instructions on configuring 
+authentication for that Provider. 
+{{< /hint >}}
 
-```yaml
-apiVersion: database.aws.crossplane.io/v1beta1
-kind: RDSInstance
+<!-- vale write-good.TooWordy = NO -->
+<!-- allow multiple -->
+ProviderConfig objects apply to individual Managed Resources. A single
+Provider can authenticate with multiple users or accounts through 
+ProviderConfigs.
+<!-- vale write-good.TooWordy = YES -->
+
+Each account's credentials tie to a unique ProviderConfig. When creating a
+managed resource, attach the desired ProviderConfig. 
+
+For example, two AWS ProviderConfigs, named 
+{{<hover label="user" line="4">}}user-keys{{</hover >}} and
+{{<hover label="admin" line="4">}}admin-keys{{</hover >}}
```

Comment:
I'm not really suggesting a change here, but FWIW two ProviderConfigs _could_ share the same secret/credentials. It would be pretty uncommon, but if the ProviderConfig exposed other configuration options you could imagine those options (not credentials) being where two ProviderConfigs varied.

Reply:
Good to know. My goal with this example is to draw attention to ProviderConfigs being applied through the managed resource, meaning that one provider can use multiple auth methods.